dev/NavigationView/NavigationViewItemAutomationPeer.h

```diff
@@ -23,10 +23,17 @@ class NavigationViewItemAutomationPeer :
     // IAutomationPeerOverrides3
     int32_t GetPositionInSetCore();
     int32_t GetSizeOfSetCore();
+    int32_t GetLevelCore();
+    void RaiseExpandCollapseAutomationEvent(winrt::ExpandCollapseState newState);
 
     // IInvokeProvider
     void Invoke();
 
+    // IExpandCollapseProvider
+    winrt::ExpandCollapseState ExpandCollapseState();
```

Comment:
>ExpandCollapseState [](start = 11, length = 19)

It should be in NavigationViewItemDataAutomationPeer just like what we did on treeview.